README.txt

```diff
@@ -399,6 +399,21 @@ popd
 echo Failed.
 exit /b 1
 
+Windows Installer using WIX Toolset
+-----------------------------------
+
+Requirements: 
+	* Visual Studio 2017
+	* Wix Toolset
+
+In Visual Studio 2017 Installer select "optional Features" -> "Wix Toolset"
+
+Build the binaries like described above, but make shure you unselect "RUN_IN_PLACE".
+"RUN_IN_PLACE" builds are not designed for installations and leads to errors if installd on newer Windows since XP!
```

Comment:
\* make sure
`and leads to errors.` (shorten and remove "installed" misspelling)